src/nyc_trees/apps/core/urls.py

```diff
@@ -4,16 +4,11 @@
 from __future__ import division
 
 from django.conf.urls import patterns, url
-from django.views.generic import TemplateView
 
-from django_tinsel.decorators import route
+from apps.core.routes import js_settings
 
 
-# These URLs have the prefix 'quiz/'
 urlpatterns = patterns(
     '',
-    url(r'^js-settings/$',
-        route(GET=TemplateView.as_view(template_name='core/settings.js',
-                                       content_type='application/javascript')),
-        name='js_settings'),
+    url(r'^js-settings/$', js_settings, name='js_settings'),
```

Comment:
We've been using it for a while: https://github.com/azavea/nyc-trees/blob/develop/src/nyc_trees/apps/core/templates/base.html#L59